src/Model/Form.php

```diff
@@ -0,0 +1,105 @@
+<?php
+
+namespace Opstalent\CrudBundle\Model;
+
+use Opstalent\CrudBundle\Traits\AvailableActionTrait;
+
+/**
+ * Class Form
+ * @author Szymon Kunowski <szymon.kunowski@gmail.com>
+ * @package Opstalent\CrudBundle
+ */
+class Form
+{
+    use AvailableActionTrait;
+
+    /**
+     * @var string
+     */
+    protected $name;
+
+    /**
+     * @var string
+     */
+    protected $action = '';
+
+    /**
+     * @var FieldModel[]
+     */
+    protected $fields = [];
+
+    /**
+     * @param string $name
+     * @return $this
+     */
+    public function setName(string $name): Form
+    {
+        $this->name = $name;
+        return $this;
+    }
+
+    /**
+     * @return string
+     */
+    public function getName(): string
+    {
+        return $this->name;
+    }
+
+    /**
+     * @param string $action
+     * @return Form
+     */
+    public function setAction(string $action): Form
+    {
+        $this->isActionAvailable($action);
```

Comment:
and this is what is sending to me "warning" when I'm reading code :smiley: 

`is...` method should return boolean. is it?

I guess it is throwing some kind of exception, but... it shouldn't